src/scripts/main.js

```diff
@@ -1 +1,67 @@
 'use strict';
+
+const successEl = (message) =>
+  `<div data-qa="notification" class="success">${message}</div>`;
+const errorEl = (message) =>
+  `<div data-qa="notification" class="error">${message}</div>`;
+
+const displayMessage = (message) => {
+  const body = document.querySelector('body');
```

Comment:
You are querying the 'body' element every time you call the 'displayMessage' function. This could be inefficient if the function is called multiple times. It would be better to query the 'body' element once and store it in a variable outside the function.